cmd/backup-manager/app/util/backup_size.go

```diff
@@ -199,6 +199,12 @@ func calcBackupSize(ctx context.Context, volumes map[string]string, level string
 func calculateSnapshotSize(volumeId, snapshotId string) (uint64, uint64, error) {
 	var snapshotSize uint64
 	var numApiReq uint64
+
+	start := time.Now()
+
+	klog.Infof("start to calculate snapshot size for %s, base on snapshot %s, volume id %s",
```

Comment:
the number of placeholder is more than the number of values.

Reply:
fixed.